Formula/mingw-w64.rb

```diff
@@ -0,0 +1,219 @@
+class MingwW64 < Formula
+  desc "Minimalist GNU for Windows with GCC."
+  homepage "https://mingw-w64.org/"
+  # https://sourceforge.net/p/mingw-w64/wiki2/Cross%20Win32%20and%20Win64%20compiler/
+  url "https://downloads.sourceforge.net/project/mingw-w64/mingw-w64/mingw-w64-release/mingw-w64-v5.0.1.tar.bz2"
+  sha256 "9bb5cd7df78817377841a63555e73596dc0af4acbb71b09bd48de7cf24aeadd2"
+
+  option "without-i686", "Compile without i686 compiler"
+  option "without-x86_64", "Compile without x86_64 compiler"
+  # option "without-multilib", "Compile x86_64 copmiler without multilib support"
```

Comment:
Currently, compile both `x86_64` toolchain and `i686` toolchain. I made the option to disable either one in case someone is building from source, since each toolchain build is time consuming